packages/edit-site/src/components/page-switcher/index.js

```diff
@@ -103,6 +112,12 @@ export default function PageSwitcher( {
 							value={ activePage.path }
 							onSelect={ onPageSelect }
 						/>
+						<LinkControl
```

Comment:
`LinkControl` seems like a weird Component to use here, since it's basically just acting as a combobox. I see how it makes sense functionally, but the name feels off. Not sure there's any action item there, just a thought.

Reply:
I think it conveys it's purpose pretty well. It's a control for links.

Reply:
👍 Should we use the same thing for Pages? (Either way, it doesn't really matter if we are introducing a new design for this in the future.)

Reply:
There are a lot fewer pages than posts. A scrollable/paginatable list will be a better fit.

Reply:
I imagine search might still be useful there in the future for large sites